services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -75,6 +75,7 @@ import {
   WorkflowRuntimeDataRepository,
 } from './workflow-runtime-data.repository';
 import mime from 'mime';
+import { env } from '@/env';
```

Comment:
To my understanding `configService` now uses `env` under the hood. See if you can get `configService` to work here.